cgi-bin/ljlib.pl

```diff
@@ -54,6 +54,14 @@ BEGIN
     }
 }
 
+# Now set up logging support for everybody else to access
+use Log::Log4perl;
```

Comment:
Right, running `bin/checkconfig.pl` pre-push is usually part of the process. (Or should be, and any new module is added there.)